### PR TITLE
fix: artist loading skeleton

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [DIA-161] 

### Description
This PR fixes the artist loading skeleton. I extract as well the dimensions logic to make sure it matches the on o nthe artist header.

## Before

https://github.com/artsy/eigen/assets/11945712/e2aa0951-891b-4ad6-a0a4-b4a7c5cf743f

___

## After

https://github.com/artsy/eigen/assets/11945712/8b655434-c7d3-4d2b-909e-24e647e15998

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- fix artist loading skeleton - mounir

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[DIA-157]: https://artsyproduct.atlassian.net/browse/DIA-157?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[DIA-161]: https://artsyproduct.atlassian.net/browse/DIA-161?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/Artist/Artist.tsx [hunk 1]
```diff
@@ -312,12 +317,16 @@ const LoadingPage: React.FC<{}> = ({}) => {
 }
 
 const ArtistSkeleton: React.FC = () => {
+  const { height, width } = useArtistHeaderImageDimensions()
```
Comment (olerichter00) [inline src/app/Scenes/Artist/Artist.tsx:320]: Nice 🌟

**Review by olerichter00 — APPROVED** [review]

File: src/app/Scenes/Artist/Artist.tsx [hunk 2]
```diff
@@ -312,12 +317,16 @@ const LoadingPage: React.FC<{}> = ({}) => {
...
+
   return (
     <Screen>
       <Screen.Header rightElements={<ShareIcon width={23} height={23} />} />
       <Screen.Body fullwidth>
         <Skeleton>
           <Flex px={2}>
+            <PlaceholderBox width={width} height={height} />
```
[the remark this answers is not in the document]
Comment (MounirDhahri) [reply src/app/Scenes/Artist/Artist.tsx:328]: thanks for pointing to this, I am unfamiliar with them but I see it used above, I will update it.

[the remark this answers is not in the document]
Comment (MounirDhahri) [reply src/app/Scenes/Artist/Artist.tsx:328]: I will also add a not to the older componnent

[the remark this answers is not in the document]
Comment (damassi) [reply src/app/Scenes/Artist/Artist.tsx]: 🙏

**Pull request merged**